cli/printers/table_printer.py

```diff
@@ -0,0 +1,123 @@
+from typing import List, Dict
+
+import click
+from texttable import Texttable
+
+from cli.consts import LICENSE_COMPLIANCE_POLICY_ID, \
+    PACKAGE_VULNERABILITY_POLICY_ID, PREVIEW_DETECTIONS_COMMON_HEADERS, SEVERITY_COLUMN, LICENSE_COLUMN, UPGRADE_COLUMN, \
+    REPOSITORY_COLUMN
+from cli.models import DocumentDetections, Detection
+from cli.printers.base_printer import BasePrinter
+
+
+class TablePrinter(BasePrinter):
```

Comment:
where you call to this class?

Reply:
On ResultsPrinter

`results_printer.py`